medicinal_plants/lib/src/common/input-fields/medicinal_upload_image.dart

```diff
@@ -0,0 +1,22 @@
+import 'package:flutter/cupertino.dart';
+
+import '../constants/color/medicinal_colors.dart';
+
+class MedicinalUploadImage {
+	static medicinalUploadImage({required double width, required double height, required double size}) {
```

Comment:
why static?
just create it like a normal widet

class MedicinalUploadImage{
   required double width,
   required double height,
   reuireqd double iconSize
   
   widget build chuchuchu
}